lib/clearsale/authentication.rb

```diff
@@ -0,0 +1,37 @@
+require 'rubygems'
+require 'multi_json'
+require 'json'
+
+module Clearsale
+  class Authentication
+    attr_accessor :token
+    def self.connector
+      @connector = ::Clearsale::Connector.new('auth/login')
```

Comment:
still same issue as mentioned in previous PR https://github.com/sellect/clearsale/pull/1#discussion_r119103724